python/lsst/ts/Dome/dome_csc.py

```diff
@@ -68,9 +72,10 @@ def __init__(
         simulation_mode=0,
         mock_port=None,
     ):
-        schema_path = (
-            pathlib.Path(__file__).resolve().parents[4].joinpath("schema", "Dome.yaml")
-        )
+        if simulation_mode not in (0, 1):
```

Comment:
This is being deprecated in salobj 6, the new way is described [here](https://ts-salobj.lsst.io/v/develop/salobj_cscs.html#simulation-mode).

Reply:
Thanks for noticing that. I am afraid there is a catch 22 here. The current development docker image still uses salobj 5 which is also specified in the version history. If I change this now then the unit tests will fail, both locally and in Jenkins. However, making the change now would prepare it for salobj6 but there is no (easy) way for me to test that the change will work.
Any idea when salobj 6 will be released? Would you accept it if I promised to make the change you suggested as soon as that release is done? I will create a new Jira ticket for it.

Reply:
The develop-env:develop is running the develop branch of salobj so that should be fine for jenkins although I see your Jenkinsfile is running salobj:develop.

Reply:
Changes made and tested against develop-env:develop. I also made sure that Jenkins uses that docker image.